cpp/CMakeLists.txt

```diff
@@ -195,6 +195,7 @@ add_library(cudf
     src/groupby/sort/group_argmin.cu
     src/groupby/sort/aggregate.cpp
     src/groupby/sort/group_collect.cu
+    src/groupby/sort/group_collect_merge.cu
```

Comment:
I don't see any direct kernel launches in `group_collect_merge.cu` can this file instead by c++ file?

Reply:
Thanks for reminding me about this, Robert. This is WIP and not complete yet, I want to have a quick working prototype for other people in my team to have a look and I will add more implementation to this file later.